### DISCUSSION TITLE
Connect multiple crazyflies via USB

### DISCUSSION THREAD

**Discussion by imreburn** [bitcraze/discussions#859] [Q&A] [answered]
Hi,

I am testing P2P communication among 2 or more crazyflies 2.1. Depending on the message that a crazyflie receives, it will print something via `DEBUG_PRINT()`. So I can see it via console (I used the example with cflib). Currently, I can connect only 1 crazyflie with the address `usb://0/` and I want to check outputs from other crazyflies. I tried something like `usb://0/E7E7E7E7E0` but it didn't work. I can't use a crazyradio as I use P2P connection. Is there way to connect 2 or more crazyflies with 1 host via USB? If not, I will try to run multiple VMs. Thank you!

**Comment by knmcguire** [comment, marked as answer]
Hi! Have you tried `usb://1/E7E7E7E7E0` for the second crazyflie? The first number in this uri indicates usually the different mediums

**Reply by imreburn** [reply]
Thank you for the reply.
It worked! I executed two cflib scripts one with URI `usb://0` and another one with `usb://1`. It seems that USB connection does not use the last 5 bytes address.

**Reply by knmcguire** [reply]
Ah no probably not :) but good it works! Yes indeed the uri afterwards make no sense haha 😅
